### PR TITLE
 Misc: Update isXML to recognize HTML-embedded elements

### PR DESCRIPTION
**Pull request by gibson042**
Fixes gh-378

### CODE REVIEW

File: src/sizzle.js [hunk 1]
```diff
@@ -559,10 +560,13 @@ support = Sizzle.support = {};
  * @returns {Boolean} True iff elem is a non-HTML XML node
```
Comment (timmywil) [inline src/sizzle.js]: Maybe we could go ahead and fix this typo, iff you want.

Comment (gibson042) [reply src/sizzle.js]: What typo? Do you think we should replace "[iff](https://en.wikipedia.org/wiki/iff)" with "if and only if"?

Comment (timmywil) [reply src/sizzle.js]: I didn't know that's what it meant.

Comment (mgol) [reply src/sizzle.js]: BTW, while I've used this notation repeatedly I'd guess it's not common knowledge among programmers without theoretical background. Expanding that could increase intelligibility of the comment to the general public.

**Pull request closed without merging**